packages/material-ui/src/styles/createSpacing.js

```diff
@@ -32,7 +32,11 @@ export default function createSpacing(spacingInput = 8) {
   }
 
   const spacing = (...args) => {
-    warning(args.length <= 4, `Too many arguments, expected [1, 4], got ${args.length}`);
+    warning(args.length <= 4, `Too many arguments, expected between 1 and 4, got ${args.length}`);
```

Comment:
nit: we accept between 0 and 4 now. Good batch PR candidate @oliviertassinari  ?

Reply:
Oops! Revised the wording before I fixed the issue :D